### PR TITLE
Update Classifier Script for Alternative Gene Prediction

### PR DESCRIPTION
**Pull request by gwaygenomics**
* Update RAS/NF1 pipeline script
* Update pancancer_classifier.py main script
    * Change wording from `tissue` to `disease`
    * Update logic in alternative gene prediction
        * This is the main update
* Update tcga_util script from `tissue` to `disease`

### CODE REVIEW

File: scripts/pancancer_classifier.py [hunk 1]
```diff
@@ -303,31 +308,33 @@
...
-
-tissue_auroc = {}
-for tissue, metrics_val in tissue_metrics.items():
+# disease specific performance
+disease_metrics = {}
+for disease in diseases:
+    sample_sub = y_matrix[y_matrix.cohort == disease].index.values
+
+    y_disease_train = y_train[y_train.index.isin(sample_sub)]
+    y_disease_pred_train = y_pred_train[y_train.index.isin(sample_sub)]
```
Comment (tj8901nm) [inline scripts/pancancer_classifier.py]: What's the difference between `y_train` and `y_pred_train`?

File: scripts/pancancer_classifier.py [hunk 2]
```diff
@@ -14,20 +14,20 @@
     --folds             number of cross validation folds (defaults to 5)
     --drop              drop the input genes from the X matrix
     --copy_number       optional flag to supplement copy number to define Y
-    --filter_count      int of low count of mutation to include tissue
-    --filter_prop       float of low proportion of mutated samples per tissue
+    --filter_count      int of low count of mutation to include disease
```
Comment (tj8901nm) [inline scripts/pancancer_classifier.py]: I noticed that for many parameters here, you set a default value below but did not specify it here.

Comment (gwaygenomics) [inline scripts/pancancer_classifier.py]: that's a good suggestion - I should specify all default parameters here for the user. Updated in next commit!

File: scripts/pancancer_classifier.py [hunk 3]
```diff
@@ -102,52 +101,54 @@
...
+base_folder = os.path.join('classifiers', genes_folder)
 
 if alt_folder != 'Auto':
     base_folder = alt_folder
 
 if not os.path.exists(base_folder):
     os.makedirs(base_folder)
-    os.makedirs(os.path.join(base_folder, 'tissue'))
 else:
     warnings.warn('Classifier may have already been built!', category=Warning)
```
Comment (gwaygenomics) [inline scripts/pancancer_classifier.py]: nice! yeah, I should and will

File: scripts/pancancer_classifier.py (hunk 1 above)
Comment (gwaygenomics) [inline scripts/pancancer_classifier.py]: `y_train` is the true labels, `y_pred_train` are the predicted labels. I'll add a comment and tweak the variable names a bit to make this more clear

**Pull request merged**